responses/test_responses.py

```diff
@@ -1141,7 +1141,7 @@ def run():
         resp = requests.get("http://example.com/two")
         assert_response(resp, "two")
         resp = requests.get("{}/one".format(httpserver.url))
-        assert_response(resp, "one")
+        assert_response(resp, "OK")
```

Comment:
This changing seems like a breaking change. This specific URL has a mock response that is now being ignored.

Reply:
That's the intention here, right? The test asserts that the response being served is the live response and not the mock response.

This shows that the passthru overrides the registered mock.

Reply:
Sure, but this will break behavior for existing users of responses which isn't something I'm willing to do at this point in time.